client/src/main/java/io/pravega/client/tables/impl/TableSegment.java

```diff
@@ -52,7 +52,7 @@
      * Inserts new or updates existing Table Entries into this Table Segment.
      *
      * @param entries A Collection of entries to insert or update. If for at least one such entry,
-     *                {@link TableEntry#getKey()#getVersion()} returns a non-null value, this will perform an atomic
+     *                {@link TableEntry#getKey()}#getVersion() returns a non-null value, this will perform an atomic
```

Comment:
Same as earlier.

Reply:
changed to `{@link TableEntry#getKey()}{@link TableKey#getVersion()}`